Apps/Sandcastle/gallery/Scene Rendering Performance.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
```

Comment:
Is there any screenshot or graphic that could be used for this Sandcastle example?